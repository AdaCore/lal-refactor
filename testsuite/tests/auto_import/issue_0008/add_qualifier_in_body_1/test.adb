package body Test is
   procedure Foo is
   begin
      Put_Line ("Foo");
   end Foo;
end Test;
