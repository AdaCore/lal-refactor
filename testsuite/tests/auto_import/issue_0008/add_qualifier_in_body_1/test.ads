with Ada.Strings.Unbounded.Text_IO;
package Test is
   procedure Foo;
end Test;
