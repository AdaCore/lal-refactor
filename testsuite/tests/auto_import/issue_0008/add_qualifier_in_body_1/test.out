Available Imports:
Import: Ada.Strings.Unbounded.Text_IO; Qualifier: Ada.Strings.Unbounded.Text_IO
Import: Ada.Strings.Wide_Unbounded.Wide_Text_IO; Qualifier: Ada.Strings.Wide_Unbounded.Wide_Text_IO
Import: Ada.Strings.Wide_Wide_Unbounded.Wide_Wide_Text_IO; Qualifier: Ada.Strings.Wide_Wide_Unbounded.Wide_Wide_Text_IO
Import: Ada.Text_IO; Qualifier: Ada.Text_IO
Import: Ada.Text_IO.Unbounded_IO; Qualifier: Ada.Text_IO.Unbounded_IO
Import: Ada.Wide_Text_IO; Qualifier: Ada.Wide_Text_IO
Import: Ada.Wide_Text_IO.Wide_Unbounded_IO; Qualifier: Ada.Wide_Text_IO.Wide_Unbounded_IO
Import: Ada.Wide_Wide_Text_IO; Qualifier: Ada.Wide_Wide_Text_IO
Import: Ada.Wide_Wide_Text_IO.Wide_Wide_Unbounded_IO; Qualifier: Ada.Wide_Wide_Text_IO.Wide_Wide_Unbounded_IO
Import: GNAT.IO; Qualifier: GNAT.IO
Import: System.IO; Qualifier: System.IO
Import: Text_IO; Qualifier: Text_IO

Edits:
test.adb
4:7-4:7 Ada.Strings.Unbounded.Text_IO.

